src/components/victory-chart/helper-methods.js

```diff
@@ -19,7 +19,7 @@ export default {
     };
 
     if (axisComponents.dependent.length === 0 && axisComponents.independent.length === 0) {
-      return [defaultAxes.independent, defaultAxes.dependent].concat(childComponents);
+      return childComponents.concat([defaultAxes.independent, defaultAxes.dependent]);
```

Comment:
Panning though a data set that crosses the y axis sometimes obscures values on that axis, particularly for area charts. It's arguably more useful to always see the corresponding values. Was there a reason for the axes being rendered under data components?